qa/full-cluster-restart/src/javaRestTest/java/org/elasticsearch/upgrades/QueryBuilderBWCIT.java

```diff
@@ -93,8 +93,8 @@ public QueryBuilderBWCIT(@Name("cluster") FullClusterRestartUpgradeStatus upgrad
 
     static {
         addCandidate("""
-            "match": { "text_field": "value"}
-            """, new MatchQueryBuilder("text_field", "value"));
+            "match": { "keyword_field": "value"}
+            """, new MatchQueryBuilder("keyword_field", "value"));
```

Comment:
Did you change this so we use "keyword_field" anywhere? Doesn't seem related to the span query stuff otherwise or am I missunderstanding anything?

Reply:
I just wanted to keep at least a query against a keyword field, to make sure its terms extraction works. I am assuming that was the purpose of having keyword, text and long field in this test.

Reply:
That ended up causing problems due to different parsing on different versions, I opted for always targeting the text field and removing the keyword field from the mappings.